### ISSUE TITLE
bug: doesn't work with pnpm

### ISSUE THREAD

**Issue by solomon23** [infi-pc/locatorjs#120]
### Where the bug happens

- [ ] Browser Extension
- [X] Library
- [ ] Other

### your browser

chrome

### Describe the bug

For the life of me I can't get this to install with PNPM

```
➜  git:(main) ✗ pnpm add @locator/babel-jsx
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In : "@locator/runtime@workspace:^" is in the dependencies but no package named "@locator/runtime" is present in the workspace

This error happened while installing the dependencies of @locator/babel-jsx@0.4.1
```

**Comment by wootra** [comment]
duplicated issue. Please add it under https://github.com/infi-pc/locatorjs/issues/119.

**Comment by wootra** [comment]
this issue is fixed on 0.4.2.
